sdk/communication/AzureCommunicationChat/.jazzy/AzureCommunicationChat.yml

```diff
@@ -2,8 +2,8 @@ author: Microsoft
 author_url: https://azure.github.io/azure-sdk/
```

Comment:
This file can actually be deleted. We no longer put files in `.jazzy` because our version update scripts don't find them. I'll address this in autorest.swift.